src/main/scala/ru/org/codingteam/horta/core/Core.scala

```diff
@@ -228,8 +228,9 @@ object Core {
   private def getCommandsDescription(pluginDefinitions: List[(ActorRef, PluginDefinition)]) =
     pluginDefinitions.map(t => t._2.name -> t._2.commands.map(cd => cd.name -> cd.level)).toMap
 
-  private def getStorages(pluginDefinitions: List[(ActorRef, PluginDefinition)]): Map[String, DAO] = {
-    pluginDefinitions.map(_._2).filter(_.dao.isDefined).map(definition => (definition.name, definition.dao.get)).toMap
+  private def getStorages(pluginDefinitions: List[(ActorRef, PluginDefinition)]): Map[String, RepositoryFactory] = {
+    pluginDefinitions.map(_._2).filter(_.repositoryFactory.isDefined).map(
```

Comment:
Several calls of `.map` in a row is discouraged and may lead to unexpected behaviour.